### PR TITLE
fix: fix config mapping, default filters

### PR DESCRIPTION
**Pull request by nandor-magyar**
Testing revealed some issues, here are the fixes forged with @m8vago .

While testing templates found some issues:
- exposing strategy selection was not visible
- default instance view was filtered (ninja fields)

@robot9706 @polaroi8d please check!

### CODE REVIEW

File: web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId]/instance/[instanceId].tsx [hunk 1]
```diff
@@ -185,7 +187,7 @@ const InstanceDetailsPage = (props: InstanceDetailsPageProps) => {
 
           {getViewStateButtons()}
         </div>
-        {viewState === 'editor' && <ImageConfigFilters onChange={setFilters} initialBaseFilter="common" />}
+        {viewState === 'editor' && <ImageConfigFilters onChange={setFilters} initialBaseFilter="all" />}
```
Comment (polaroi8d) [inline web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId]/instance/[instanceId].tsx:190]: Why?

File: web/crux-ui/src/pages/products/[productId]/versions/[versionId]/image/[imageId].tsx [hunk 2]
```diff
@@ -194,7 +194,7 @@ const ImageDetailsPage = (props: ImageDetailsPageProps) => {
 
           {getViewStateButtons()}
         </div>
-        {viewState === 'editor' && <ImageConfigFilters onChange={setFilters} />}
+        {viewState === 'editor' && <ImageConfigFilters onChange={setFilters} initialBaseFilter="all" />}
```
Comment (polaroi8d) [inline web/crux-ui/src/pages/products/[productId]/versions/[versionId]/image/[imageId].tsx:197]: Why?

**Review by polaroi8d — CHANGES_REQUESTED** [review]
If @m8vago is a co-author please add his name to the PR description body.

File: web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId]/instance/[instanceId].tsx (hunk 1 above)
Comment (nandor-magyar) [reply web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId]/instance/[instanceId].tsx:190]: TL;DR When I was testing as a **user** hidden fields in deployments were confusing.

I thought the we have bugs, but the fields that were in use were simply just hidden. On an image screen, it is perfectly fine to have hidden fields, but for a deployment it is obscure, everything that has some values, therefore will be deployed, should be visible. Templates have values that can get lost into the dark.
What do you think?

Comment (polaroi8d) [reply web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId]/instance/[instanceId].tsx:190]: Talked in person, okay for me.

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**